lib/fhir_client/client.rb

```diff
@@ -170,10 +170,10 @@ def capability_statement(format = @default_format)
     # Method returns a conformance statement for the system queried.
     # @return
     def conformance_statement(format = @default_format)
-      if @cached_conformance.nil? || format != @default_format
+      if @cached_capability_statement.nil? || format != @default_format
```

Comment:
I think we have the `attr_accessor`, so we probably could drop the `@` here (and a couple other places), but not the end of the world either way.